### PR TITLE
Rename some task names

### PR DESCRIPTION
**Pull request by pfliu-nlp**

### CODE REVIEW

File: explainaboard/tasks.py [hunk 1]
```diff
@@ -130,7 +134,7 @@ class TaskCategory:
         ],
     ),
     TaskCategory(
-        "kg-link-tail-prediction",
+        "kg_prediction",
```
Comment (neubig) [inline explainaboard/tasks.py:137]: I know this is WIP, but I was looking at it out of curiosity and noticed that this name uses underbars while other names use hyphens. This was also the case in DataLab as well. Maybe we can consolidate to hyphens?

Comment (pfliu-nlp) [reply explainaboard/tasks.py:137]: Yes, this is a typo, and thanks for capturing it!

**Comment by pfliu-nlp** [comment]
Sister PR: https://github.com/ExpressAI/DataLab/pull/199

**Review by neubig — APPROVED** [review]
Looks good, thanks!

**Pull request merged**